app/forms/draft_work_form.rb

```diff
@@ -103,7 +103,7 @@ def deserialize_license(params)
              populator: ContributorPopulator.new(:authors, Author),
              prepopulator: ->(*) { authors << Author.new if authors.blank? },
              on: :work_version,
-             &contributor
+             &author
 
```

Comment:
this doesn't seem right to me, the `authors` collection references the contributor -- this would fix it